mkdocs.yml

```diff
@@ -54,7 +54,8 @@ theme:
 nav:
     - How To:
         - Get Started with Besu:
-            - System Requirements: HowTo/Get-Started/System-Requirements.md
+            - System Requirements Private Networks: HowTo/Get-Started/System-Requirements-Private.md
+            - System Requirements Public Networks: HowTo/Get-Started/System-Requirements-Public.md
```

Comment:
I would suggest making the title reversed for faster reading and search engines indexing:
- System Requirements Private Networks ->  Private Networks System Requirements
- System Requirements Public Networks -> Public Networks System Requirements

Reply:
Fixed